### ISSUE TITLE
Questions about learning rate and batch size setting on training superpoint

### ISSUE THREAD

**Issue by brade31919** [rpautrat/SuperPoint#71]
Hi @rpautrat ,

First, thank you for the great work!
I have some quick questions about the setting in configs/superpoint_coco.yaml.
According to the paper, they used same learning rate and batch size in training both magic point and superpoint (lr=0.001, batch_size=32).
Is there any reason that you chose lr=0.0001 and batch_size=3 instead of using the original setting? Or you found that results were worse using the original setting?

Thank you!

**Comment by rpautrat** [comment]
Hi,
Yes, I found out that the training of SuperPoint was much more stable with a lower learning rate than the original one. The batch size reduced to 3 was mainly to be able to fit a batch in the memory of my GPU. A higher batch size would result in an Out Of Memory error for me.

**Comment by brade31919** [comment]
Understood!
I also encountered memory usage problem even on a gpu cluster ... 
Thank you for your information!!
